src/legacy/core_plugins/data/public/expressions/expressions_service.ts

```diff
@@ -71,23 +70,23 @@ export interface Interpreter {
   interpretAst(ast: Ast, context: Context, handlers: Handlers): Promise<Result>;
 }
 
-type InterpreterGetter = () => Promise<{ interpreter: Interpreter }>;
-
-export interface ExpressionsServiceDependencies {
-  interpreter: {
-    renderersRegistry: RenderFunctionsRegistry;
-    getInterpreter: InterpreterGetter;
-  };
+interface InterpreterGetter {
+  getInterpreter: () => Promise<{ interpreter: Interpreter }>;
 }
 
 /**
  * Expressions Service
  * @internal
  */
 export class ExpressionsService {
```

Comment:
A bit unrelated but shouldn't we have `registerFunction` here?
I saw @ppisljar added it to expressions on markdown vis.